exercises/crypto-square/.meta/solutions/crypto_square.rb

```diff
@@ -9,6 +13,7 @@ def normalize_plaintext
   end
 
   def plaintext_segments
+    return [] if normalize_plaintext == ''
```

Comment:
The canonical data had a test which asserts that an empty array passed into ```Crypto.plaintext_segments``` will return an empty array. However, the current ```Crypto``` class just blows up when it tries to call ```each_slice``` with a size of 0.